acme/client.go

```diff
@@ -670,6 +670,7 @@ func (c *Client) requestCertificateForCsr(authz []authorizationResource, bundle
 
 			break
 		default:
+			defer resp.Body.Close()
```

Comment:
@miekg Because it might be read in line 621. If that's never the case, then sure, but I don't want to break things. I need feedback from @xenolf.